builtin/providers/aws/resource_aws_codecommit_trigger.go

```diff
@@ -0,0 +1,170 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codecommit"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsCodeCommitTrigger() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsCodeCommitTriggerCreate,
+		Read:   resourceAwsCodeCommitTriggerRead,
+		Delete: resourceAwsCodeCommitTriggerDelete,
+
+		Schema: map[string]*schema.Schema{
+			"repository_name": &schema.Schema{
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Required: true,
+				ValidateFunc: func(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
As this is just a reference to a repository, should we validate here? Isn't the validation in the `aws_code_commit` resource? :)